test/es6/objlit.js

```diff
@@ -15,15 +15,58 @@ var tests = [
         }
     },
     {
-        name: "Method shorthand",
+        name: "Shorthand names `get` and `set` parse without error (they are not keywords in these cases)",
+        body: function () {
+            var a = 0;
+            var get = 1;
+            var set = 2;
+            var z = 3;
+
+            var o = { get };
+            var p = { set };
+            var q = { get, set };
+            var r = { set, get };
+            var s = { get, z };

```

Comment:
Correct.  I added a new test for `get` and `set` name use in object destructuring to `test/es6/destructuring_obj.js`